src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
```

Comment:
It's more efficient to create a single instance of Random and reuse it, rather than creating a new instance every time the method is called. Consider making it a private class member.